skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/services/TjenestebussIntegrasjonService.kt

```diff
@@ -0,0 +1,361 @@
+            }
+    }
+
+    suspend fun bestillBrev(
+        call: ApplicationCall,
+        bestillBrevRequestDto: BestillBrevRequestDto
+    ): ServiceResult<BestillBrevResponseDto.Success, BestillBrevResponseDto.Failure> {
+        return tjenestebussIntegrasjonClient.post(call, "/bestillbrev") {
+            Domain.BestillBrevRequestDto(
+                brevKode = bestillBrevRequestDto.brevKode,
+                brevGruppe = bestillBrevRequestDto.brevGruppe,
+                isRedigerbart = bestillBrevRequestDto.isRedigerbart,
+                sprakkode = bestillBrevRequestDto.sprakkode,
+                sakskontekstDto = Domain.SakskontekstDto(
+                    journalenhet = bestillBrevRequestDto.sakskontekstDto.journalenhet,
+                    gjelder = bestillBrevRequestDto.sakskontekstDto.gjelder,
+                    dokumenttype = bestillBrevRequestDto.sakskontekstDto.dokumenttype,
+                    dokumentdato = bestillBrevRequestDto.sakskontekstDto.dokumentdato,
+                    fagsystem = bestillBrevRequestDto.sakskontekstDto.fagsystem,
+                    fagomradekode = bestillBrevRequestDto.sakskontekstDto.fagomradekode,
+                    innhold = bestillBrevRequestDto.sakskontekstDto.innhold,
+                    kategori = bestillBrevRequestDto.sakskontekstDto.kategori,
+                    saksid = bestillBrevRequestDto.sakskontekstDto.saksid,
+                    saksbehandlernavn = bestillBrevRequestDto.sakskontekstDto.saksbehandlernavn,
+                    saksbehandlerId = bestillBrevRequestDto.sakskontekstDto.saksbehandlerId,
+                    sensitivitet = bestillBrevRequestDto.sakskontekstDto.sensitivitet
+                )
+            )
+        }.toServiceResult<Domain.BestillBrevResponseDto.Success, Domain.BestillBrevResponseDto.Failure>()
+            .map {
+                BestillBrevResponseDto.Success(
+                    journalpostId = it.journalpostId
+                )
+            }.catch { error ->
+                BestillBrevResponseDto.Failure(message = error.message, type = error.type)
+            }
+    }
+
+    suspend fun redigerDokument(
+        call: ApplicationCall,
+        journalpostId: String,
+        dokumentId: String,
+    ): ServiceResult<RedigerDokumentResponseDto.Success, RedigerDokumentResponseDto.Failure> {
+        return tjenestebussIntegrasjonClient.post(call, "/hentSamhandler") {
+            Domain.RedigerDokumentRequestDto(journalpostId = journalpostId, dokumentId = dokumentId)
+        }.toServiceResult<Domain.RedigerDokumentResponseDto.Success, Domain.RedigerDokumentResponseDto.Failure>()
+            .map {
+                RedigerDokumentResponseDto.Success(
+                    metaforceURI = it.metaforceURI
+                )
+            }.catch { error ->
+                RedigerDokumentResponseDto.Failure(message = error.message, type = error.type)
+            }
+    }
+
+
+    private object Domain {
```

Comment:
Glemte du å slette noe som du flytta til andre filer? :) SamhandlerTypeCode er definert to steder.